legal-api/tests/unit/services/warnings/business/business_checks/test_corps.py

```diff
@@ -0,0 +1,13 @@
+# Copyright © 2022 Province of British Columbia
```

Comment:
2024

Reply:
Will do this on the next commit (Test cases)

Reply:
Done.